samples/apps/autogen-studio/pyproject.toml

```diff
@@ -28,8 +28,9 @@ dependencies = [
     "python-dotenv",
     "websockets",
     "numpy < 2.0.0",
+    "sqlmodel"
 ]
-optional-dependencies = {web = ["fastapi", "uvicorn"]}
+optional-dependencies = {web = ["fastapi", "uvicorn"], database = ["psycopg2"]}
```

Comment:
Can psycopg (psycopg3) be used instead?

Reply:
Thanks. It seems the recommended approach is to specify `psycopg` which will use the latest version. I'll do just that.